pythondata_cpu_vexriscv/verilog/src/main/scala/vexriscv/GenCoreDefault.scala

```diff
@@ -6,10 +6,12 @@ import spinal.lib._
 import spinal.lib.sim.Phase
```

Comment:
I don't understand why this is changing a scala file? Shouldn't this be part of the source module?

Reply:
This is the top generator script that handles user options and then builds the VexRiscv accordingly, adding the plugins etc..   It's not part of the VexRiscv "library".   It is local to the pythondata_cpu_vexriscv repo.

For example, cache line size is currently hardcoded in this file.  If we want to make it an option that can be changed, we'd alter this file to add an user option and then pass the value to where the caches are instantiated.